wsgi.py

```diff
@@ -50,6 +51,15 @@ def post_adviser_result():  # Ignore PyDocStyleBear
     adapter = AdvisersResultsStore()
     adapter.connect()
     document_id = adapter.store_document(request.json)
+    if(request.form.get("origin")):
+        url = request.form.get("origin")
+        service = url.split("/")[2].split(".")[0]
+        OpenShift.schedule_kebechet(
```

Comment:
Follow this pattern please:

https://github.com/thoth-station/user-api/blob/d5fdb89f854cac971846ddafb5c0d242539b7cb1/thoth/user_api/api_v1.py#L57

and then run that on the instance:

```
_OPENSHIFT.<_method_name>(...)
```